lib/signal-manager.js

```diff
@@ -0,0 +1,44 @@
+const runningProcs = new Set()
+
+const forwardedSignals = [
+  'SIGINT',
+  'SIGTERM'
+]
+
+function handleSignal (signal) {
```

Comment:
Style nit: slightly preferred to do `const handleSignal = signal => {` rather than `function handleSignal (signal) {`, because it makes it clear that the function cannot potentially be used as a class or method.